Source/items.cpp

```diff
@@ -1388,7 +1388,7 @@ void CreatePlrItems(int p)
 		{
 			SetPlrHandItem(&plr[p].HoldItem, IDI_WARRCLUB);
 			GetPlrHandSeed(&plr[p].HoldItem);
-			AutoPlace(p, 0, 1, 3, TRUE);
+			AutoPlaceItemInInventory(p, plr[p].HoldItem, true);

```

Comment:
What's happening here?

Reply:
I hadn't seen this comment. Do you still have any questions about it?

Basically, we just remove the hardcoded grid position and let the function take care of placing the item in the inventory using the standard iteration logic.

Reply:
Yeah i eventually figured it out when going over things by commit :P